README.md

```diff
@@ -159,6 +159,29 @@ SteelSeries.GameSense.GSClient.Instance.SendEvent("HEALTH", currentHealth);
 + Remember that you can specify script execution order in your project. Make sure your setup script runs before anything that calls `SendEvent()`
 + If you use the client through script, make sure you do not instantiate `GameSenseManager.prefab` in your scene hierarchy. Also, do not attach any client's scripts to any game objects.
 
+# Preparing Image Data
+GameSense for OLED/LCD allows you to specify bitmaps as sources for frame data. This section details the options available to you to accomplish this.
+
+## Texture Assets
+It is more than likely that freshly imported texture assets will not work out of the box with GameSense for OLED/LCD. To ensure the client is able to process the assets, follow these steps:
+
+1. In project browser, navigate to the directory containing your image files and select those you'd like to use.
+2. In the Inspector, locate and set the following properties to the corresponding values:
+   +  sRGB (Color Texture) = unchecked
+   +  Read/Write Enabled = checked
+   + Generate Mip Maps = unchecked
+   + Compression Settings = _RGBA 32 bit_
+
+3. Click ***Apply***
+
+The texture assets now have the correct format and can be used as image sources for frames data in screen handlers. Simply use `Texture2D Image Data` asset and set the texture property in the inspector or call `ImageDataTexture2D.Create()` from script. For best results, ensure the textures have correct size for the target screen device.
+
+## Bit Vector
+You can supply a byte array where the value of each bit corresponds to a white or black point on the screen as described [here](https://github.com/SteelSeries/gamesense-sdk/blob/master/doc/api/json-handlers-screen.md#showing-raw-bitmaps). It is mandatory for the length of the array to be equal to `ceiling(`_`width`_ `*` _`height`_ `/ 8)`, where _width_ and _height_ are the dimensions of the target screen device. See `ImageDataBitVector` class.
+
```

Comment:
I'm not really clear on how I can supply a byte array to be used.  Where do I find this `ImageDataBitVector` class?

Reply:
> I'm not really clear on how I can supply a byte array to be used. Where do I find this `ImageDataBitVector` class?

Here it is: [click me](https://github.com/sharkgoesmad/unity-gamesense-client/blob/ce138f439af6359351db8d295c12d72dcac89399/Types/FrameDataImage.cs#L47)